cmd/down.go

```diff
@@ -68,24 +68,25 @@ func Down() *cobra.Command {
 				if err != nil {
 					return err
 				}
-			}
+			} else {
+				devName := ""
+				if len(args) == 1 {
+					devName = args[0]
+				}
+				dev, err := utils.GetDevFromManifest(manifest, devName)
+				if err != nil {
+					return err
+				}
 
-			devName := ""
-			if len(args) == 1 {
-				devName = args[0]
-			}
-			dev, err := utils.GetDevFromManifest(manifest, devName)
-			if err != nil {
-				return err
-			}
+				if err := runDown(ctx, dev, rm); err != nil {
+					analytics.TrackDown(false)
+					err = fmt.Errorf("%w\n    Find additional logs at: %s/okteto.log", err, config.GetAppHome(dev.Namespace, dev.Name))
+					return err
+				}
 
-			if err := runDown(ctx, dev, rm); err != nil {
-				analytics.TrackDown(false)
-				err = fmt.Errorf("%w\n    Find additional logs at: %s/okteto.log", err, config.GetAppHome(dev.Namespace, dev.Name))
-				return err
+				analytics.TrackDown(true)
```

Comment:
I think this has to be outside the else, right? I mean, we want to track the command execution even when the flag `--all` is specified, right?